kube_apiserver_metrics/assets/dashboards/overview.json

```
@@ -0,0 +1,809 @@
+{
+  "title": "Kubernetes API Server - Overview",
+  "description": "",
+  "widgets": [
+    {
+      "id": 5523550815136180,
+      "definition": {
+        "type": "image",
+        "url": "/static/images/screenboard/integrations/kubernetes.jpg",
+        "sizing": "cover",
+        "has_background": true,
```

a cosmetic nit:to not shrink the logo, the option `scale-down` can be used

```suggestion
        "sizing": "scale-down",
        "has_background": false,
```

---

Added in new commit https://github.com/DataDog/integrations-core/pull/9757/commits/1dfe81546c72df3eb79370fee014767b74f526b3 👍